### PR TITLE
Perl pdl

### PR DESCRIPTION
**Pull request by wlmb**
## Description
Solution with Perl/PDL using bytes for bits instead of default doubles

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by wlmb** [comment]
Using byte PDL's instead of the default the speed increases about 50%.

**Comment by wlmb** [comment]
I added a solution_2 employing the PDL::PP pp_def mechanism of PDL that allows writing C-like threadable functions. Its speed is about 30Xperl's solution_1.

File: PrimePDL/solution_2/primes.pl [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    my $bits   = $self->{bits};
+    my $factors=zeroes(long,$q+1-3)->xvals*2+3;
+    say $factors if $DEBUG;
+    $bits->run_sieve_aux($factors);
+}
+
+sub print_results {
+    my ( $self, $show_primes, $show_stats, $duration, $passes ) = @_;
+    say $self->get_primes if $show_primes;
+    printf "Luis_Mochán_(wlmb)_Perl/PDL;%d;%f;%d;algorithm=base,faithful=yes,bits=8\n",
```
Comment (rbergen) [inline PrimePDL/solution_2/primes.pl]: Please use a different label for this solution than the one used for solution_1. I'd offer `Luis_Mochán_(wlmb)_Perl/PDL-PP`  as a suggestion, but as before you're free to deviate.

Comment (wlmb) [reply PrimePDL/solution_2/primes.pl]: Sure!

**Pull request merged**